sympy/printing/tests/test_latex.py

```diff
@@ -1984,7 +1975,7 @@ def test_translate():
 def test_other_symbols():
     from sympy.printing.latex import other_symbols
     for s in other_symbols:
-        assert latex(symbols(s)) == "\\"+s
+        assert latex(symbols(s)) == r"" "\\" + s
```

Comment:
Is this a typo?

Reply:
It is not. I just wanted to be consistent and use a raw string in that scenario too, however, `r"\"` is invalid in Python. See https://docs.python.org/3/faq/design.html#why-can-t-raw-strings-r-strings-end-with-a-backslash.

Reply:
Oh yes, I forgot about that. Still I don't think that the empty rawstring is needed though if it isn't actually doing anything. If we have to use `"\\"` then that's fine.

Reply:
That's right, we don't need that `r""` bit, but for the sake of consistency I would include it. I also have it here: https://github.com/sympy/sympy/pull/20915/files#diff-d909b0940fe23d80682ae4319261f194ca445b99359428612db7f13ae8da7d2cR299. Let me know if you want this to be modified (either here or in both places) and I will update the PR.